labkey/query.py

```diff
@@ -294,13 +300,17 @@ def insert_rows(
     :param query_name: table name to insert into
     :param rows: set of rows to insert
     :param container_path: labkey container path if not already set in context
+    :param options: Additional parameters
     :param timeout: timeout of request in seconds (defaults to 30s)
     :return:
     """
     url = server_context.build_url("query", "insertRows.api", container_path=container_path)
 
     payload = {"schemaName": schema_name, "queryName": query_name, "rows": rows}
 
+    if options is not None:
+        payload = payload | options
```

Comment:
I think we should use the alternate syntax for this because our docs say we support Python 3.7+, and the `|` operator was introduced in 3.9. I'm not totally against requiring a newer version, but that can often be a big ask for IT teams, so I really only want to do that if we feel like we're getting other advantages from requiring a newer version.

```suggestion
        payload = {**payload, **options}
```